Sources/SwiftSignalRClient/WebSocket.swift

```diff
@@ -599,10 +600,14 @@ private class InnerWebSocket: Hashable {
         ws.binaryType = binaryType
         return ws
     }
-
+    
+    #if swift(>=5.0)
     func hash(into hasher: inout Hasher) {
         hasher.combine(id)
     }
+    #else
+    var hashValue: Int { return id }
```

Comment:
Can this be removed completely? XCode was complaining about this even before changing the version to 5 so I am wondering if we can only keep the `hash(into:)` version and avoid language switch.

Reply:
If using the source with anything less than swift 5, the `Hashable` protocol will probably be expecting this to be implemented. Something else I can test.